posthog/templates/shared_resource_404.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        {% include "head.html" %}
+    </head>
+    <style>
+        html, body, iframe {
```

Comment:
It would be cool if you could load our font using `@font`, will look more professional compared to the Times New Roman thing we have going on